src/test/resources/mapping/expected_marc_record.mrc

```diff
@@ -1 +1 @@
-00080nam a2200037 a 450024500420000000aThe Journal of ecclesiastical history
\ No newline at end of file
+00147nam a2200061 a 4500001000200000245004200002999004100044100aThe Journal of ecclesiastical historyffic8b50e3f-0446-429c-960e-03774b88223f
```

Comment:
Looks like this is an invalid marc record
![Screen Shot 2020-04-21 at 9 32 26 AM](https://user-images.githubusercontent.com/41596468/79871875-084efb80-83b3-11ea-81f2-62300f95bd88.png)

Reply:
I will check, thanks

Reply:
looks good now